site/pages/api/delete-user.api.ts

```diff
@@ -0,0 +1,20 @@
+import { NextApiResponse } from "next";
+import { deleteAdminUser } from "../../data/cognito";
+import { DeleteUserApiRequest } from "../../interfaces";
+import { redirectTo, redirectToError } from "../../utils/apiUtils";
+
+const deleteUser = async (req: DeleteUserApiRequest, res: NextApiResponse): Promise<void> => {
+    try {
+        const { username } = req.body;
+
+        await deleteAdminUser(username);
```

Comment:
Thinking about it we probably need to be more defensive here, we should verify that the user being deleted has the same orgId as the admin doing the deleting, we can grab the logged in user's orgId by calling `getSession` and then we probably need a new function in cognito.ts to get a user's details by their username using `AdminGetUserCommand`

Reply:
done